### PR TITLE
More unit overview fixes

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Bunch of smallish bug fixes for things we found during bug bash.

Each bug is a separate commit.

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -368,8 +368,7 @@
   "unnamedFunction": "You have a variable or function that does not have a name. Don't forget to give everything a descriptive name.",
   "unattachedBlockTipTitle": "Unattached blocks",
   "unattachedBlockTipBody": "Blocks that are not attached will not do anything. If you want these blocks in your program, try connecting them to other blocks.",
-  "unplugged": "UNPLUGGED",
```
Comment (joshlory) [inline apps/i18n/common/en_us.json:371]: Not needed?

File: dashboard/app/views/scripts/show.html.haml [hunk 2]
```diff
@@ -44,14 +44,6 @@
...
       %br/
 
-    - if @script.has_lesson_plan?
-      %a{href: CDO.code_org_url('/curriculum/' + @script.name)}
-        = t('home.lesson_plans')
-      %br/
-      %br/
-
-    %br/
-
```
Comment (joshlory) [inline dashboard/app/views/scripts/show.html.haml:54]: This removal will also affect users who aren't opted into the experiment, correct?  Will that be an issue?

Comment (Bjvanminnen) [reply dashboard/app/views/scripts/show.html.haml:54]: Yeah, this is intentional.

File: apps/i18n/common/en_us.json (hunk 1 above)
Comment (Bjvanminnen) [reply apps/i18n/common/en_us.json:371]: Yeah, the usage was removed in an earlier PR and I had neglected to remove the string.

**Comment by joshlory** [comment]
"Lesson" string change will probably have Eyes test impact as well (for next dev-of-day).

**Pull request merged**